packages/client/src/Dashboard.tsx

```diff
@@ -371,11 +373,25 @@ function ConversationRow({
   dispatch: (action: Sel.Action) => void
   navigate: RouteComponentProps["navigate"]
 }) {
-  const { from, date, id, isRead, snippet, subject, isStarred } = conversation
+  const {
+    from,
+    date,
+    id,
+    isRead,
+    snippet,
+    subject,
+    labels,
+    isStarred
+  } = conversation
+
+  let newLabels = labels
+  newLabels = newLabels && newLabels.map(label => label.slice(1))
```

Comment:
Declarations with `let` are often a code smell. In this case you can combine this into one line and declare with `const`.

Another issue is that `newLabels` is not a descriptive name. Maybe the name could be `labelsForDisplay`?

The `slice` call removes the leading backslash, right? The problem is that custom labels do not begin with a backslash. (See https://developers.google.com/gmail/imap/imap-extensions#access_to_gmail_labels_x-gm-labels) A more robust solution would be to use the `replace` method with a regular expression argument. But this is display logic, and I would rather put display logic as late as possible at the point where a value is actually displayed, which would be in `DisplayLabel`.